crates/e2e/src/subxt_client.rs

```diff
@@ -109,8 +109,9 @@ where
     C: subxt::Config,
     E: Environment,
 {
-    api: ReviveApi<C, E>,
-    contracts: ContractsRegistry,
+    // TODO: make private once call builder supports RLP
```

Comment:
We should start adding our handles behind the `TODO`, so that we don't get confused who's working on what.

Reply:
good idea. https://github.com/use-ink/ink/pull/2375/commits/c2364c43f2272268aecf92cb5739f4b796e11b1d